### PR TITLE
E2e journey

### PR DESCRIPTION
**Pull request by zielinskikamil**
Journey: 
Create Session & Listing with different case type, 
Find Them in session search page,
Assign to each other,
Verify that dialogue with a problem is displayed
Check if visible in a calendar together with the listing request

### CODE REVIEW

File: e2e/pages/calendar.po.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  private calendarEntryClass = by.className('fc-v-event');
+
+  getNumberOfVisibleEvents(): promise.Promise<number> {
+    this.quickWaitToRenderEvents();
+    return element.all(this.calendarEntryClass).count();
+  }
+
+  quickWaitToRenderEvents() {
+    // Due some reasons events in calendar aren't displayed until some action will be taked
+    // workaround for it is to select already selected view mode (month/week/day/list)
```
Comment (leszekgu) [inline e2e/pages/calendar.po.ts]: might be connected to the initialized binding in our calendar component, that it is fired before ucCalendar is assigned, but I may be wrong here.
The thing is, that this issue seemed to be resolved long time ago.

File: e2e/pages/listing-creation.po..ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { ElementHelper } from './../utils/element-helper';
```
Comment (leszekgu) [inline e2e/pages/listing-creation.po..ts]: why there are two dots in this file name ? ;)

File: e2e/utils/element-helper.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+export class ElementHelper {
+  async clear(elem: ElementFinder, length?: number) {
+    const inputText = await elem.getAttribute('value');
+    length = length || inputText.length || 100;
+    let backspaceSeries = '';
+    for (let i = 0; i < length; i++) {
+      backspaceSeries += Key.BACK_SPACE;
+    }
+    elem.sendKeys(backspaceSeries);
```
Comment (leszekgu) [inline e2e/utils/element-helper.ts]: Wouldn't it be better to clear it and then send one backspace?

Comment (zielinskikamil) [reply e2e/utils/element-helper.ts]: that's the case here that `.clear()` not always work, solution for it is to send series of backspace.

**Pull request merged**